test/scripts/website-comparisons.test.ts

```diff
@@ -0,0 +1,71 @@
+import { test } from 'tap';
+import {
+  checkWebsiteAndReturnHash,
+  cleanWebsiteData,
+  isPdf,
+} from '../../scripts/lib/website-comparisons';
+
+test('Only HTML body and readable content is returned', tap => {
+  const unclean_html: string =
+    "<html> <head> <meta content='Incorrect to be shown'> <link as='style' href='/doc/assets/stylesheets/fonts-b49e8aae.css' rel='stylesheet'> <meta content='ie=edge' http-equiv='X-UA-Compatible'> </head> <body>Correct content<div> and div contents</div> </body> </html>";
+  tap.equal(cleanWebsiteData(unclean_html), 'Correct content and div contents');
+  tap.end();
+});
+
+test('If no body is present, and no readable html, an empty string is returned', tap => {
+  const unclean_html: string =
+    "<html> <head> <meta content='Incorrect to be shown'> <link as='style' href='/doc/assets/stylesheets/fonts-b49e8aae.css' rel='stylesheet'> <meta content='ie=edge' http-equiv='X-UA-Compatible'> </head></html>";
+  tap.equal(cleanWebsiteData(unclean_html), '');
+  tap.end();
+});
+
```

Comment:
Add a test for the no-body-but-readable-html case?

Reply:
Done